lib/auth/join_kubernetes.go

```diff
@@ -43,7 +43,7 @@ func (a *Server) checkKubernetesJoinRequest(ctx context.Context, req *types.Regi
 
 	userInfo, err := a.kubernetesTokenValidator.Validate(ctx, req.IDToken)
 	if err != nil {
-		return trace.Wrap(err)
+		return trace.WrapWithMessage(err, "failed to validate the Kubernetes token")
```

Comment:
funny, I've never seen `trace.WrapWithMessage` before, probably since regular `trace.Wrap` lets you include a message already. I guess it avoids 2 checks when you already know `err != nil && len(args) > 0`. TIL